### ISSUE TITLE
Learning Stereo from Single Images 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#53]
# TL;DR
In depth estimation, they propose a method to generate stereo image pairs from monocular images and use them for stereo depth estimation model training. First, the monocular depth estimation model is used to estimate depth, and then the opposite image is estimated. Then both left and right images are plugged into a stereo depth estimation model. The accuracy improvement of various stereo depth estimation models is confirmed.
![image](https://user-images.githubusercontent.com/24524018/91037622-48193e80-e644-11ea-9a95-3e877e12b16f.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.01484

# Submission Dates(yyyy/mm/dd)
2020/08/04

# Authors and institutions
Jamie Watson, Oisin Mac Aodha, Daniyar Turmukhambetov, Gabriel J. Brostow, Michael Firman
- Niantic 
- University of Edinburgh 
- UCL 

# Methods
![image](https://user-images.githubusercontent.com/24524018/91037628-4bacc580-e644-11ea-8f12-17ed42f2c7f5.png)

# Results

# Comments
ECCV2020
